src/components/Home.test.js

```diff
@@ -0,0 +1,15 @@
+import React from 'react';
+import { render } from '@testing-library/react';
+import { MemoryRouter } from 'react-router-dom';
+import Home from './Home';
+
+describe('Home', () => {
+  it('should render correctly', () => {
+    const { container } = render(
+      <MemoryRouter>
+        <Home />
+      </MemoryRouter>,
+    );
+    expect(container).toMatchSnapshot();
+  });
```

Comment:
- [ ] Please make sure that you fix this failing test that is the test for your home component :pray: 
  
  ![image](https://github.com/CarlosIgreda/Math-Magicians/assets/92303891/30a2843a-5628-417e-a926-c4e0616c495a)

Reply:
Hi @Owusu-Desmond, thanks for your feedback. Required changes are done.

![image](https://github.com/CarlosIgreda/Math-Magicians/assets/116421768/a8040547-8a9d-4e60-8e36-d2d0a7afeec2)